.github/workflows/post-release.yaml

```diff
@@ -47,6 +47,7 @@ jobs:
     needs: release
     if: needs.release.outputs.type == 'latest'
     uses: gravitational/teleport/.github/workflows/update-ami-ids.yaml@master
+    secrets: inherit
```

Comment:
Can you do the same for `update-docs-version` too?

(I recognize this is another PR to master, but at least you could include the backport along with this PR)

Reply:
The `update-docs-version` job does not call another workflow so it should not need this `secrets: inherit` clause. All the steps in that job will have access to the secrets already.

Reply:
Ok, I see what you mean now that I've seen this job run (and fail). The `app_id` uses `vars.APP_ID` - vars are from environments and the workflow/job/step do not specify an environment to use. It looks like we have a "post-release" environment and it seems that's what we want to use here. It defines the `APP_ID` var and the `PRIVATE_KEY` secret.

https://github.com/gravitational/teleport/pull/30890 raised to fix this.